scripts/Events.ts

```diff
@@ -0,0 +1,111 @@
+import { EventDBManager } from "../test/EventDBManager";
+import { ethers } from "hardhat";
+import type { UniswapV3Pool2 } from "../typechain";
+
+enum EventType {
+  MINT = 1,
+  BURN = 2,
+  SWAP = 3,
+}
+
+let liquidityEventDB: EventDBManager;
+let swapEventDB: EventDBManager;
+let poolAddress = "0x8ad599c3A0ff1De082011EFDDc58f1908eb6e6D8";
+let fromBlock = 12777394;
+let toBlock = 13556212; //13556212;
+let batchSize = 500;
+
+async function main() {
+  liquidityEventDB = await EventDBManager.buildInstance(
+    "liquidity_events_usdc_weth_3000.db"
+  );
+  swapEventDB = await EventDBManager.buildInstance(
+    "swap_events_usdc_weth_3000.db"
+  );
+  let uniswapV3Pool = (await ethers.getContractAt(
+    "UniswapV3Pool2",
+    poolAddress
+  )) as UniswapV3Pool2;
+
+  while (fromBlock <= toBlock) {
+    let endBlock =
+      fromBlock + batchSize > toBlock ? toBlock : fromBlock + batchSize;
+    await saveEvents(EventType.MINT, fromBlock, endBlock);
+    await saveEvents(EventType.BURN, fromBlock, endBlock);
+    // await saveEvents(EventType.SWAP, fromBlock, endBlock);
```

Comment:
So we are not saving the swaps into the db? Can we also parameterize this?

Reply:
No problem. And this is because fromblock is different for liquidity and swap events in our latest database, so we need to handle them separately.